settings.gradle

```diff
@@ -1,6 +1,6 @@
 pluginManagement {
     gradle.ext.kotlinVersion = '1.8.21'
-    gradle.ext.agpVersion = '7.2.1'
+    gradle.ext.agpVersion = '7.3.1'
```

Comment:
⚠️ As discussed internally (p1686309448014639-slack-CC7L49W13), for now, we won't upgrade AGP. Hence, we should revert this change.